go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-azurerm
 
 require (
-	github.com/Azure/azure-sdk-for-go v30.0.0+incompatible
+	github.com/Azure/azure-sdk-for-go v30.1.0+incompatible
```

Comment:
this is an SDK upgrade, we do these as separate PRs. Could we revert this change?